uid-pages/layout-living-application/src/resources/page.properties

```diff
@@ -1,7 +1,7 @@
 #The name must start with 'custompage_'
 #Mon Feb 25 17:37:11 CET 2019
 name=custompage_defaultlayout
-displayName=Default living application layout
-description=This is the default layout V5 definition for a newly-created application. It was created using the UI designer, so you can export it and edit it with the UI designer. It contains a horizontal menu widget and an iframe widget. The menu structure  is defined in the application navigation. The application pages are displayed in the iframe.
```

Comment:
Maybe "This layout is deprecated. Use the Bonita Layout instead."

Reply:
done